### PR TITLE
feat: about pages (new IA)

### PR DESCRIPTION
**Pull request by agnieszkastanczuk**
## Description 📝

New About pages.

## Related Issues 🔗

JIRA issue: [FSU-382](https://klau.atlassian.net/browse/FSU-382)

## Preview Links 🔍
[Fundacja](https://sterczaceuszy-git-about-pages-sterczace-uszy.vercel.app/fundacja)
[Dane-finansowe](https://sterczaceuszy-git-about-pages-sterczace-uszy.vercel.app/dane-finansowe)

### CODE REVIEW

File: packages/frontend/app/templates/About/About.jsx [hunk 1]
```diff
@@ -6,32 +6,27 @@ import Achievements from "./components/Achievements"
 import AboutUs from "./components/AboutUs"
 import Values from "./components/Values"
 import Supporters from "./components/Supporters"
+import Hero from "./components/Hero"
```
Comment (imklau) [inline packages/frontend/app/templates/About/About.jsx:9]: @agnieszkastanczuk take a look at Github Actions / CI warnings, it looks like there are some variables that you don't use 👀 

Zrzut ekranu 2023-11-26 o 20 21 50

File: packages/frontend/app/templates/About/About.jsx [hunk 2]
```diff
@@ -40,12 +35,9 @@ const About = ({
           animalsNumber={animals.total}
           adoptedAnimalsNumber={adoptedAnimals.total}
         />
-        <FinancialStatements data={financialStatements.items} />
       </Container>
-
-      <Microchips />
-
-      <Supporters data={supporting.items} />
+      {/* <Microchips />
```
Comment (imklau) [inline packages/frontend/app/templates/About/About.jsx:39]: some comments left 🕵🏼

File: packages/frontend/app/templates/FinancialData/FinancialData.jsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+import Hero from "./components/Hero"
+import Page from "../Page"
+
+import { financialStatementsType } from "../../lib/types"
+
+import * as Styled from "./FinancialData.styled"
+import FinancialStatements from "./components/FinancialStatements"
+
+const FinancialData = ({ financialStatements }) => (
+  <Page title="O nas">
```
Comment (imklau) [inline packages/frontend/app/templates/FinancialData/FinancialData.jsx:13]:
```suggestion
  <Page title="Dane finansowe">
```

**Review by imklau — COMMENTED** [review]
Thank you for updating these pages 🙏🏼 ❤️ 
Look really nice, I left some small comments though

File: packages/design-system/blocks/navigation/data.js [hunk 4]
```diff
@@ -1,7 +1,16 @@
 const data = [
   {
     title: "Fundacja",
-    href: "/fundacja",
+    links: [
+      {
+        title: "Fundacja",
```
Comment (imklau) [inline packages/design-system/blocks/navigation/data.js:6]:
```suggestion
        title: "O nas",
```

The href is fine, but the title of this link should be "O nas"

**Review by imklau — APPROVED** [review]

**Review by paulgrym — APPROVED** [review]

**Pull request merged**